src/Native/System.Security.Cryptography.Native/pal_cms.h

```diff
@@ -61,3 +61,49 @@ extern "C" int CryptoNative_CmsGetRecipientStackFieldCount(CmsRecipientStack* re
 Shims the CMS_RecipientInfo_free method
 */
 extern "C" void CryptoNative_CmsRecipientDestroy(CMS_RecipientInfo* recipient);
+
+/*
+Initializes a CMS_ContentInfo with the cipher given by the oid in algorithmOid and returns a pointer
+to it. 
+
+Status is set to 1 on success, 0 on unsupported algorithm, and -1 on null input.
+*/
+extern "C" CMS_ContentInfo* CryptoNative_CmsEncryptInitialize(ASN1_OBJECT* algorithmOid, int32_t* status);
+
+/*
+Adds all the originator certificates to the CMS_ContentInfo structure.
```

Comment:
"all" Does that mean it can only be called once?

Does this require an enveloped CMS?  What happens if we pass a future SignedCms to it?